app/js/lib/foxbox.js

```diff
@@ -40,54 +40,82 @@ let themes = [
 
 const _private = {
   isPollingEnabled: Symbol('isPollingEnabled'),
-  nextPollTimeout: Symbol('nextPollTimeout')
+  nextPollTimeout: Symbol('nextPollTimeout'),
+
+  fetchServices: Symbol('fetchServices')
 };
 
 /**
- * Request a JSON from a specified URL.
+ * Request a content of the specified type from a specified URL.
  *
- * @param {string} url The URL to send the request to.
- * @param {string} method The HTTP method (defaults to "GET").
- * @param {!Object} body An object of key/value.
+ * @param {String} url The URL to send the request to.
+ * @param {String} contentType The content mime type (eg. image/jpeg).
+ * @param {String} [method=GET] The HTTP method (defaults to "GET").
```

Comment:
I usually use the annotations types of [Closure Compiler](https://developers.google.com/closure/compiler/docs/js-for-compiler#types) where the type `@param {string}` is lower case (all types are lower case except `Object` and `Array`).